src/components/ToggleBtn.jsx

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export default function ToggleBtn({ buttonText, buttonHandler }) {
+  return <StyledBtn onClick={buttonHandler}>{buttonText}</StyledBtn>;
+}
+
+const StyledBtn = styled.button`
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
```

Comment:
```suggestion
  padding: 0.5rem 1rem;
```
으로 단축으로 줄일 수도 있을 것 같아요!(이건 선택의 차이이겠네요)